AlphaWallet/ScrollableSegmentedControl/ScrollableSegmentedControlAdapter.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  ScrollingSegmentedControlAdapter.swift
+//  AlphaWallet
+//
+//  Created by Jerome Chan on 28/12/21.
+//
+
+import UIKit
+
+class ScrollableSegmentedControlAdapter: ScrollableSegmentedControl, ReusableTableHeaderViewType {
+    static var reusableIdentifier: String {
```

Comment:
> Seems to work as expected. Is this PR going to replace the segment bars used in other parts of the app like when importing a wallet (not sure if there are more)?
> 
> What is the reason between `ScrollableSegmentedControlAdapter` and `ScrollableSegmentedControl`. What does it adapt the latter to?
> 
> IMO, the video isn't necessary here. It's helpful, but a screenshot would probably cost you less time and still point out to the reviewer which part of the app is affected.

No, this PR only replaces this one instance. The adapter just makes this a drop in replacement for this instance. There are some differences in set up. I wanted to see if this would have minimal disruption so that when we do have to replace it everywhere it would be easier. If you check the TokensViewController file, the only thing that is changed is the type.

Reply:
Removed `static var reusableIdentifier: String`